### DISCUSSION TITLE
Private sharing or at least remove metadata?

### DISCUSSION THREAD

**Discussion by anphex** [photoprism/photoprism#4336] [Q&A]
I want to share an album to a website but photoprism serves all my metadata, exact location, phone, everything, already below the thumbnail. Is there any way to hide or remove this? Removing keywords, labels and coordinates neither works, it's still shown.

**Comment by graciousgrey** [comment]
Thank you for your feedback! 

Hiding metadata on shared links is not yet possible. Advanced sharing options are planned for the future. We will probably also add an option that will make it possible to only allow to download images with removed metadata from shared albums.
